### PR TITLE
[C-2205] Wait for user to follow in sign up

### PR DESCRIPTION
**Pull request by sliptype**
### Description

We were rendering `HomeStack` as soon as `hasAccount` was true, meaning that if the account finished creating while the user was still on the screen to pick first follows it would automatically switch to the feed screen
* Add a couple new actions to track sign up progress on mobile (`startedSignOnProcess` was not used)
* Add a selector `getHasCompletedAccount` to designate that the user has an account and they have completed the sign up flow
* Ensure the Rate and Notification drawers don't get shown until the user has a completed account

### Dragons

Sign up changes! Dangerous

### How Has This Been Tested?

Tested:
* Sign up flow with user taking long time to follower accounts
* Sign up flow with user quickly following accounts
* Sign in

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by raymondjacobson — APPROVED** [review]
makes sense

File: packages/web/src/common/store/pages/signon/selectors.js [hunk 1]
```diff
@@ -31,6 +34,16 @@ export const getSuggestedFollowIds = (state) => {
   return categories[selectedCategory] || []
 }
 
+export const getHasCompletedAccount = createSelector(
```
Comment (dylanjeffers) [inline packages/web/src/common/store/pages/signon/selectors.js:37]: nice

**Review by dylanjeffers — APPROVED** [review]
def all makes sense to me, potentially could have a sign-up status that maps to the various phases better, but two bools is fine too.

**Comment by sliptype** [comment]
> def all makes sense to me, potentially could have a sign-up status that maps to the various phases better, but two bools is fine too.

Yeah we have the page in redux but it only gets set in web. I was thinking through updating the redux page when navigating but there were a lot of cases to catch so I kept it simple for now

**Pull request merged**